### PR TITLE
Log out branch

### PR DESCRIPTION
**Pull request by talgelfand**

### CODE REVIEW

File: src/firebase/config.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { initializeApp } from 'firebase/app'
-import { getAuth } from 'firebase/auth'
+import { getAuth} from 'firebase/auth'
```
Comment (talgelfand) [inline src/firebase/config.js]: Add a space before the second bracket please

File: src/components/Signup/Signup.js [hunk 2]
```diff
@@ -56,60 +57,61 @@ function Signup() {
...
+          />
+          <InputControl
+            label='Password'
+            type='password'
+            placeholder='Enter Password'
+            onChange={(event) => setValues((prev) => ({ ...prev, pass: event.target.value }))}
+          />
+          <br></br>
+          {/* TODO: I would suggest we make it select input */}
+          <InputControl
```
Comment (talgelfand) [inline src/components/Signup/Signup.js:78]: Just to clarify: are you going to change it to select input in another PR?

File: src/components/NavBar/NavBar.js [hunk 3]
```diff
@@ -1,16 +1,45 @@
...
+    position: 'bottom-center',
+    autoClose: 3000,
+    hideProgressBar: false,
+    closeOnClick: true,
+    pauseOnHover: true,
+    draggable: true,
+  }
+
+  const logOut = () => {
+
```
Comment (talgelfand) [inline src/components/NavBar/NavBar.js]: Please remove this empty line :)

File: src/components/NavBar/NavBar.js [hunk 4]
```diff
@@ -1,16 +1,45 @@
...
     <>
       <Nav>
         <NavLink to='/'>Home</NavLink>
-        <NavLink to='/statistics'>Statistics</NavLink>
-        <NavLink to='/login' activeStyle>Login</NavLink>
-        <NavLink to='/signup'>SignUp</NavLink>
-        {/* TODO: there's a logout function in firebase - can you implement please? */}
-        <NavLink to='/login'>LogOut</NavLink>
+        {currentUser && <NavLink to='/statistics'>Statistics</NavLink>}
+        {!currentUser &&<NavLink to='/login' activeStyle>
```
Comment (talgelfand) [inline src/components/NavBar/NavBar.js]: Can you make sure prettier works for you please? A space is needed after `&&` 🙏

File: src/components/InputControl/InputControl.style.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+`
+
+export const input = styled.div`
+  border-radius: 5px;
+  border: 1px solid #dddddd;
+  outline: none;
+  padding: 10px 15px;
+  color: #000;
+`
+
```
Comment (talgelfand) [inline src/components/InputControl/InputControl.style.js:26]: Please remove those extra empty lines

File: src/components/InputControl/InputControl.style.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components'
+
+export const container = styled.div`
```
Comment (talgelfand) [inline src/components/InputControl/InputControl.style.js]: Can you please name styled components starting with a capital letter like `Container`

File: src/components/InputControl/InputControl.style.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+export const label = styled.div`
+  font-weight: 700;
+  font-size: 1rem;
+  color: #313131;
+`
+
+export const input = styled.div`
+  border-radius: 5px;
+  border: 1px solid #dddddd;
+  outline: none;
```
Comment (talgelfand) [inline src/components/InputControl/InputControl.style.js:18]: I would probably add an outline to inputs so it's clear to user on which input they currently are

File: src/components/InputControl/InputControl.module.css [hunk 8]
```diff
@@ -23,6 +23,3 @@
   border-color: #ccc;
```
Comment (talgelfand) [inline src/components/InputControl/InputControl.module.css]: This file can be removed if you've rewritten it to styled components

File: src/components/InputControl/InputControl.js [hunk 9]
```diff
@@ -1,6 +1,5 @@
-// TODO: not sure this component is neede - might as well be replaced with Bootstrap component
 import React from 'react'
-
+// import { container, label } from './InputControl.style'
```
Comment (talgelfand) [inline src/components/InputControl/InputControl.js]: Why? 😃

File: src/components/InputControl/InputControl.js [hunk 10]
```diff
@@ -1,6 +1,5 @@
...
 import styles from './InputControl.module.css'
 
 function InputControl(props) {
```
Comment (talgelfand) [inline src/components/InputControl/InputControl.js]: Can you use `const` here please for consistency with other components?

File: src/components/InputControl/InputControl.js [hunk 11]
```diff
@@ -1,6 +1,5 @@
-// TODO: not sure this component is neede - might as well be replaced with Bootstrap component
 import React from 'react'
-
+// import { container, label } from './InputControl.style'
 import styles from './InputControl.module.css'
```
Comment (talgelfand) [inline src/components/InputControl/InputControl.js]: I guess it's not needed anymore?

File: src/components/InputControl/InputControl.js [hunk 12]
```diff
@@ -1,6 +1,5 @@
-// TODO: not sure this component is neede - might as well be replaced with Bootstrap component
```
Comment (talgelfand) [inline src/components/InputControl/InputControl.js:1]: Tbh I doubt this component is needed because we can just the one from Bootstrap. Might be a separate PR though

File: src/components/Signup/Signup.js [hunk 13]
```diff
@@ -56,60 +57,61 @@ function Signup() {
...
+            label='Gender'
+            placeholder='Enter your gender'
+            onChange={(event) => setValues((prev) => ({ ...prev, gender: event.target.value }))}
+          />
+          {/* TODO: I would suggest we make it calendar input */}
```
Comment (talgelfand) [inline src/components/Signup/Signup.js:83]: Are you going to change it to a calendar picker in another PR?

File: src/components/Signup/Signup.js (hunk 2 above)
Comment (iljasem58) [reply src/components/Signup/Signup.js:78]: Will do it later :)

File: src/components/InputControl/InputControl.js (hunk 9 above)
Comment (iljasem58) [reply src/components/InputControl/InputControl.js]: In progress 👯‍♂️
